_posts/en/newsletters/2021-09-08-newsletter.md

```diff
@@ -0,0 +1,211 @@
+through the wallet and client interfaces are validated and relayed using the
+mempool."
+  a4link="https://bitcoincore.reviews/22340#l-97"
+
+  q5="What is the difference between blocksonly and block-relay-only? Should
+these changes be applied for block-relay-only connections as well?"
+  a5="Blocksonly mode is a node setting, while block-relay-only is an attribute
+of a peer connection. When a node is started in blocksonly mode, the node sends
+`fRelay=false` in the version handshake with all peers and disconnects peers that
+send any transaction-related messages. Whether or not in blocksonly mode, a node
+may have block-relay-only connections on which they ignore incoming transaction
+and addr messages. As such, the existence of block-relay-only connections has no
+relation to a node's mempool contents and ability to reconstruct blocks from
+compact block messages, so these changes shouldn't be applied to
+block-relay-only connections."
+  a5link="https://bitcoincore.reviews/22340#l-111"
+%}
+
+## Preparing for taproot #12: Vaults with taproot
+
+*A weekly [series][series preparing for taproot] about how developers
+and service providers can prepare for the upcoming activation of taproot
+at block height {{site.trb}}.*
+
+{% include specials/taproot/en/11-vaults-with-taproot.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 22.0rc3][bitcoin core 22.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software. Major changes in this new
+  version include support for [I2P][topic anonymity networks] connections,
+  removal of support for [version 2 Tor][topic anonymity networks] connections,
+  and enhanced support for hardware wallets.
+
+- [Bitcoin Core 0.21.2rc2][bitcoin core 0.21.2] is a release candidate
+  for a maintenance version of Bitcoin Core.  It contains several bug
+  fixes and small improvements.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #22009][] introduces a heuristic to quantify thriftiness
+  of [coin selection][topic coin selection] results. The heuristic can
+  be configured with the new option `-consolidatefeerate`, which
+  determines the boundary between consolidatory and thrifty behavior.
+  The wallet will now prefer the least wasteful input set from results
```

Comment:
@Xekyo what is "the least wasteful input set"?

Reply:
Well, we call the heuristic the "waste metric", but given that I actually removed that term in one of my iterations, I should probably go with something like:

```suggestion
  The wallet will now prefer the best scoring input set from results
```

Alternatively, I'd mention "a heuristic called the 'waste metric'" in the first sentence. @harding: Do you have a preference in either direction?

Reply:
@Xekyo at present, without re-reading your paper, both terms are equally meaningless to me.  What makes one input set less wasteful, or better scoring, than another?

Reply:
Okay, I took another stab at it, adding a bit more background information. I hope that it's clearer now, although it may be a bit too long now.

Reply:
I've broken up/rearranged the description a bit, so that it starts with the overview (use both algos and decide which is best), then describes the heuristic, then concludes with the end result and future work.